sssoja/tv-bland
# So/add cast list data

### What changes have you made?
- for consistency, the `Cast` component has now been renamed to `CastList`
- a new exportable type has been created called `CastMemberType` which is made up of an image, name, character and URL
- a new prop has been created called `castList` with a type of `CastMemberType[]`
- the `castList` will receive a `const cast` which has a value of `json._embedded ? json._embedded.cast : [];` 
- `json._embedded` represents the key to the cast array in the API


### Which issue(s) does this PR fix?
Fixes #20 

### Screenshots (if there are design changes)
<img width="245" alt="Screenshot 2020-12-04 at 16 46 50" src="https://user-images.githubusercontent.com/53219789/101184160-ae085f80-3650-11eb-911f-30728294adf7.png">


### How to test
Select a few shows from the homepage at random and check that the following data points are displayed under _Starring_:
- image 
- name
- character name 

Check for any Typescript errors in the event no data is available for any of the given data points above
